app/master/cluster_master.py

```diff
@@ -55,6 +62,22 @@ def __init__(self):
 
         SlavesCollector.register_slaves_metrics_collector(lambda: self.all_slaves_by_id().values())
 
+    def configure_heartbeat(self):
+        self._heartbeat_job = self._scheduler.add_job(self.heartbeat, 'interval', seconds=self._heartbeat_frequency)
+        return self._scheduler
+
+    def heartbeat(self):
```

Comment:
It seems like the word "heartbeat" refers to two different things. On the slave it's an API call to the master. On the master it's a periodic cleanup of the slave list. Should these have different names?